src/arsort.py

```diff
@@ -663,3 +662,24 @@ def make_dirs(argflag, fitsdict, filesort):
     else: os.mkdir(newdir)
     # Return the name of the science targets
     return sci_targs
+
+
+def set_instr_name(inst_name):
```

Comment:
I like that.  Am implementing as
mosaic camera LRISb
but I could go with shortname